diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/classes/DataClassesRule.kt

```diff
@@ -97,14 +97,14 @@ class DataClassesRule(configRules: List<RulesConfig>) : DiktatRule(
                 .none { it.elementType in badModifiers } &&
                     classBody?.getAllChildrenWithType(FUN)
                         ?.isEmpty()
-                        ?: false &&
+                    ?: false &&
```

Comment:
Does it look like expected indent?

Reply:
> Does it look like expected indent?

Yes, this is how IDEA formats _Elvis_ followed by the 2nd operand within boolean expressions, see the [tests](https://github.com/saveourtool/diktat/pull/1545/files#diff-24354b1e72b5bd9be3fd0086e869a4352fc5f39d6fd528197832c18245045390R2286) under `IndentationRuleTest.Multi-line Elvis expressions`:

```kotlin
                val elvisExpressionInsideBinaryExpressionA = true &&
                        ""
                            ?.trim()
                            ?.trim()
                            ?.trim()
                            ?.isEmpty()
                        ?: true
```